docs/changelog.rst

```diff
@@ -22,6 +22,7 @@ Fixed
 
 Deprecated
 ----------
+- Removed ``to_slippy_tiles`` and ``to_xyz_tiles`` methods from the raster module (#1110). We recommend to use the package cht_tiling instead.
```

Comment:
perhaps good to mention that because of this core can no longer _write_ vrt files even if we can still read them?